CIP--HeptaSean-DomainValidation/README.md

```diff
@@ -0,0 +1,493 @@
+---
+CIP: HeptaSean-DomainValidation
+Title: Domain Validation for Cardano Addresses
+Authors: Benjamin Braatz <bb@bbraatz.eu>
+Comments-URI: 
```

Comment:
```suggestion
Comments-URI: https://forum.cardano.org/t/cip-draft-domain-validation-for-cardano-addresses/106328
```

Reply:
already fixed (my review & your last update crossed each other)

Reply:
Already did add the forum link, before reading your suggestion. ;)